NASA-AMMOS/aerie-ui
# Simulation Templates UI

Resolves #429 

Depends on https://github.com/NASA-AMMOS/aerie/pull/851

To test template creation:
1. Navigate to a plan
1. Open the simulation panel
1. Make changes to the parameters values
3. Click on the "Simulation Template" input
4. Type in a name for the template
5. Click on the "+" icon
6. Confirm that the changed values are still what you changed them to, but now they have a blue/purple dot next to them
7. Confirm that the selected template's name is now displayed in the "Simulation Template" input

To test value revert:
1. Change the parameters again
3. Cmd/ctrl click on the orange dots for some of the changed parameters
4. Confirm that the values are reverted to either the template's values or the default values for the mission model

To test template switching:
1. Create a couple more presets following the "Template creation" steps
2. Click on the "Simulation Template" input again
3. Select a different templates from the one you currently have selected
5. Confirm that the parameter values are updated to reflect the newly selected template
